src/index.css

```
@@ -160,6 +160,11 @@
     fill: rgba(14, 165, 233, 0.15);
   }
 
+  .polygon-core {
+    fill: rgba(34, 197, 94, 0.25) !important;
```

Stylelint (17.9.0)

[error] Expected "0.25" to be "25%" (alpha-value-notation)

(alpha-value-notation)